<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <property name="windowIcon">
   <iconset resource="../../assets/main_app.qrc">
    <normaloff>:/leaf.ico</normaloff>:/leaf.ico</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="session_code_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;Righteous&quot;;
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Session Code: Fetching...</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="players_list">
   <property name="geometry">
    <rect>
     <x>436</x>
     <y>78</y>
     <width>407</width>
     <height>542</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(../assets/lobbyContainer.png);
font: 16pt &quot;Righteous&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="start_game_button">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>507</x>
     <y>642</y>
     <width>269</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Righteous</family>
     <pointsize>14</pointsize>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-image: url(../assets/buttonNormal.png);
border: none;
}

QPushButton::pressed{
background-image: url(../assets/buttonPressed.png);
border: none;
}
</string>
   </property>
   <property name="text">
    <string>START GAME</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="retry_button">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>350</y>
     <width>269</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Righteous</family>
     <pointsize>14</pointsize>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-image: url(../assets/buttonNormal.png);
border: none;
}

QPushButton::pressed{
background-image: url(../assets/buttonPressed.png);
border: none;
}
</string>
   </property>
   <property name="text">
    <string>RETRY CONNECTION</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../../assets/main_app.qrc"/>
 </resources>
 <connections/>
</ui>
